systemdfiles/mozdefweb.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=MozDef Web Interface (meteor)
+After=mongod.service
+
+[Service]
+# Requires systemd version 211 or newer
+User=mozdef
+Group=mozdef
+# Client Env Vars
+Environment=HOME="/opt/mozdef"
+Environment=MONGO_URL=mongodb://localhost:3002/meteor
+Environment=ROOT_URL=http://your.domain.name.here
+Environment=PORT=3000
+Environment=NODE_TLS_REJECT_UNAUTHORIZED="0"
+ExecStart=/bin/bash -c '/usr/bin/node /opt/mozdef/envs/meteor/mozdef/bundle/main.js > /opt/mozdef/meteor/meteor.log 2>&1 &'
```

Comment:
I'd recommend you do `/usr/bin/node /opt/mozdef/envs/meteor/mozdef/bundle/main.js` instead, you can access the logs with `journalctl -u mozdefweb`. You might have to remove `Type=forking`